docs/rancher/cloud-provider.md

```diff
@@ -44,6 +44,52 @@ When spinning up an RKE2 cluster using the Harvester node driver, select the `Ha
 
   ![](/img/v1.2/rancher/rke2-cloud-provider.png)
 
+You can also deploy the Harvester Node Driver manually. On the **Cluster Configuration** page, select `None` for **Cloud Provider** as shown below. 
+
+  ![](/img/v1.2/rancher/choose_cloud_provider_none.png)
+
+Perform the following steps to manually deploy the Harvester Node Driver.
+
+1. Generate and inject cloud-config for `harvester-cloud-provider`.
+
+:::note
+
+The script uses `kubectl` and `jq` to operate the Harvester cluster. The script needs access to the `Harvester Cluster` kubeconfig to work. The `<serviceaccount name>` is usually your guest cluster name, and the `<namespace>` needs to match the namespace of the guest cluster.
+
+:::
+
+- Install `jq`:
+  ![](/img/v1.2/rancher/install_jq.png)
+
+- Set up `kubectl` and the kubeconfig file:
+  ![](/img/v1.2/rancher/setup_kubectl_and_kubeconfig.png)
+
+You can generate the kubeconfig file using the [generate_addon.sh](https://raw.githubusercontent.com/harvester/cloud-provider-harvester/master/deploy/generate_addon.sh) script. It is available on the [harvester/cloud-provider-harvester](https://github.com/harvester/cloud-provider-harvester) repo.
+
+```
+# ./generate_addon.sh <serviceaccount name> <namespace>
+```
+1. The output of this script contains cloud-init user data similar to the sample below. Copy and paste the script output to **Machine Pools > Show Advanced > User Data**.
+
+  ![](/img/v1.2/rancher/generated_kubeconfig.png)
+
+  ![](/img/v1.2/rancher/patch_user_data.png)
+
+1. After the VM is re-provisioned, add the following `HelmChart` yaml of `harvester-cloud-provider` to **Cluster Configuration > Add-On Config > Additional Manifest**.
+
+  ![](/img/v1.2/rancher/harvester_cloud_provider_yaml.png)
+
+1. Disable the in-tree cloud provider.
+
+  - Go to **Add-On Config** and select **Edit as YAML**.
+  ![](/img/v1.2/rancher/edit-k3s-cluster-yaml.png)
+
+  - Set `disable-cloud-controller: true` to disable the default RKE2 cloud controller.
+  - Add `cloud-provider=external` to use the Harvester cloud provider.
+  ![](/img/v1.2/rancher/disable_and_setup_cloud_provider.png)
+
+With these settings configured, an RKE2 cluster should successfully provision using the external cloud provider.
+
```

Comment:
I think we should dismiss the `None` option here since use the built-in `Harvester` cloud-provider is always recommended. For the CSI driver, it would different since the cloud provider is not a required dependency.